src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Operators.cs

```diff
@@ -285,7 +285,7 @@ private bool GetStandardAndLiftedBinopSignatures(List<BinOpFullSig> rgbofs, BinO
                     case ConvKind.Identity:
                         if (cv2 == ConvKind.Identity)
                         {
-                            BinOpFullSig newsig = new BinOpFullSig(this, bos);
```

Comment:
@jkotas, @stephentoub, what is the policy for taking changes identified by analyzers for "servicing only" projects?

Reply:
Most of our "servicing only" projects are yours :wink:

We've generally been ok with it as long as the changes are minimal and obviously correct.  But we do have a few older projects where it's just not worth it and we've often suppressed such wide-reaching analyzers there as we've turned them on across the repo.

I'm fine with whatever you and @jaredpar prefer here.

Reply:
The changes to Microsoft.CSharp and System.Linq.Expressions are simple enough, so let's take the changes to those projects rather than suppressing analysis of those projects.